Makefile

```diff
@@ -1,7 +1,9 @@
 .PHONY: test docs
 
-ENABLE_BRANCH_COVERAGE ?= 0
-AUTO_FIX_IMPORTS ?= 0
+export ENABLE_BRANCH_COVERAGE ?= 0
+export AUTO_FIX_IMPORTS ?= 0
+export CONCURRENCY_BACKEND ?= eventlet
+export COVERAGE_FAIL_UNDER ?= 100
```

Comment:
I think it should be restricted to the subprocesses in the makefile (which I thought would be ok).
The parent shell (so the environment that called make) is uneffected as far as I know. 
To make sure I just tried this:

```sh
make test_lib
echo $CONCURRENCY_BACKEND
# prints an empty string
```

In a normal unix shell to get the environment of any script into the calling environment one would have to use "source" i.e. `source ./script_exporting_env_variables.sh` otherwise the environment will always be restricted to subprocesses.

Reply:
Ah great. Your refactored version is much better then. Thanks!